developer-guides/rest-api/im/list/README.md

```diff
@@ -56,11 +56,61 @@ curl -H "X-Auth-Token: 9HqLlyZOugoStsXCUfD_0YdwnNnunAJF8V47U3QHXSq" \
     ],
     "success": true
```

Comment:
Hey @xbolshe, can you add the `offset`, `count` and `total` properties to this response object please?

Reply:
It seems `"t": "p"` is incorrect for Direct Messages. I updated request/response using real Rocket Chat.